web/libs/editor/src/tags/control/Choice/Choice.scss

```diff
@@ -74,7 +74,7 @@ body :global(.ant-checkbox-wrapper) {
 
 body :global(.ant-checkbox .ant-checkbox-inner) {
   background-color: var(--color-neutral-surface);
-  border-color: var(--color-neutral-border);
+  border-color: var(--color-neutral-border) !important;
```

Comment:
How neccessary is this? I know that we will rewrite Choice for our own `@ui/checkbox` so not against this dirty fix, but just curious.

Reply:
the `antd reset` css file is applying `!important` already and cannot bypass the specificity without altering it so it is neccessary I am afraid.